tools/nni_cmd/constants.py

```diff
@@ -79,6 +79,21 @@
     'SMAC': 'smac_tuner'
 }
 
+DISPATCHERS_SUPPORTING_IMPORT_DATA = {
```

Comment:
There are only 2 dispatchers:  default and multiphase dispatcher.  Use term "Tuner" to represent tuner and advisor would be better.

Reply:
Have renamed it